### PR TITLE
Converted all the css into Tailwind css of Badge component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Converted all the css into Tailwind css of Badge component
Issue - #1725 

# Issue Resolved

Fixes #NA

### CODE REVIEW

File: components/Badge/Badge.module.css [hunk 1]
```diff
@@ -1,19 +1,6 @@
...
-  align-items: center;
-  margin: 1rem;
-}
-
 .Badge > img,
 .Badge > svg {
   fill: currentcolor;
   height: 100px;
   margin: 1rem 0;
 }
```
Comment (kylemh) [inline components/Badge/Badge.module.css:6]: https://csstailwind.com/nested-child-selector-in-tailwindcss-style-nested-elements/

**Review by kylemh — COMMENTED** [review]
almost done here

File: components/Badge/Badge.js [hunk 2]
```diff
@@ -16,15 +15,20 @@ Badge.defaultProps = {
 
 function Badge({ className, icon, isImageFirst, label }) {
   return (
-    <figure className={classNames(styles.Badge, className)}>
+    <figure
+      className={classNames(
+        'flex flex-col justify-items-center place-content-start m-2 [&>img>svg]:fill-current [&>img>svg]:my-4 [&>img>svg]:mx-0 [&>img>svg]:h-24',
```
Comment (kylemh) [inline components/Badge/Badge.js:20]:
```suggestion
        'flex flex-col justify-items-center place-content-start m-2 [&>img>svg]:fill-current [&>img>svg]:my-4 [&>img]:mx-0 [&>img]:h-24' [&>svg]:mx-0 [&>svg]:h-24',
```

**Comment by kylemh** [comment]
After you finish this PR, perhaps you could integrate Tailwind into our Storybook instance? I forgot about the need to do that!

**Comment by subhajit20** [comment]
> After you finish this PR, perhaps you could integrate Tailwind into our Storybook instance? I forgot about the need to do that!

Okay cool, in the same way I have to do that in stodybook components right?

**Comment by kylemh** [comment]
There shouldn't be any CSS modules to convert in Storybook, it's just adjusting the Storybook config so Tailwind is part of its build process. Right now, storybook has no idea what `m-1` does, for example.

**Comment by subhajit20** [comment]
> There shouldn't be any CSS modules to convert in Storybook, it's just adjusting the Storybook config so Tailwind is part of its build process. Right now, storybook has no idea what `m-1` does, for example.

Okay got it.
Would you merge this pr?

**Comment by kylemh** [comment]
I will as soon as I can! I need to resolve an issue that's visible on this PR's preview deployments on the `/services` route with the Badge components in-use. If I merge without resolving the issue, the problem goes live to prod!

There's nothing you need to do in this PR more though!

**Comment by subhajit20** [comment]
> I will as soon as I can! I need to resolve an issue that's visible on this PR's preview deployments on the `/services` route with the Badge components in-use. If I merge without resolving the issue, the problem goes live to prod!
> 
> There's nothing you need to do in this PR more though!

Alright!

**Comment by kylemh** [comment]
So, @subhajit20 I noticed a problem that you'll need to be aware of...

We've defined the element with the ID `__next` as a point for tailwind to raise its specificity from ([source](https://github.com/OperationCode/front-end/blob/main/tailwind.config.js#L25)) ([tailwind docs on the topic](https://tailwindcss.com/docs/configuration#selector-strategy)).

The downside to this approach is that we need to fixes [like this](https://github.com/OperationCode/front-end/pull/1739/commits/d371d872a8cd636c59dc7e24051d6277480ea9c2#diff-1059fc20b96c24c9675739154b47111f651ccf26b6d4cb2531c74a5ed54166e7) to all the places USING a component we do a Tailwind update to. It's lame, but at least it's formulaic and simple.

We could _not_ do `important: '#__next'` in the Tailwind config; however, if we do that we have the same problem in the other direction... and the solution seems much worse: using the [important selector](https://csstailwind.com/how-to-use-important-modifier-in-tailwindcss-complete-guide/) for all Tailwind classes we want to "win".

The benefit of changing the CSS modules files is that it's temporary, since our end goal has us deleting CSS modules files and the "winner" going forward will just be whatever classname was applied last (ideal for us).

Anyways... to look out for this, you'll want to leverage the "find all references" action in VS Code:

Screenshot 2023-09-20 at 4 55 17 PM

For every component that you work on, you'll want to look for usages, find the page they're rendered on, look at the page and compare the same UI to prod, and either update Tailwind all the way through to the implementation level OR do the fix I recommended above.

**Pull request merged**